### PR TITLE
Fix hero navs on JSX pages

### PR DESCRIPTION
**Pull request by lancesnider**
1. On small screens the `hero-nav` splits into multiple lines in strange places. 

screen shot 2019-01-25 at 3 59 14 pm

This was because each button was just a span so the browser was treating them as one long word. It would break it into multiple lines where there were word breaks within a button. 

screen shot 2019-01-25 at 2 57 19 pm

I wanted to make a `HeroNav` component, but each set of links was built in such a unique way that I don't think `HeroNav` would have added much value. Maybe a component would be nice in the future, but for now wrapping menus in `<ul>`'s does the trick. 

screen shot 2019-01-25 at 4 25 39 pm
screen shot 2019-01-25 at 4 24 39 pm
screen shot 2019-01-25 at 4 24 30 pm
screen shot 2019-01-25 at 4 24 22 pm

I also added a `focus` state so that you can see which button or `<a>` is selecting when tabbing through the page. 

screen capture on 2019-01-25 at 15-26-49

## Out of scope

I didn't fix the collections `hero-nav` menu. That page is currently built in .cjsx. I went to convert it to .jsx, but that would have required rewriting at least a couple other .cjsx files. I'm happy to do that, but it should probably be its own PR. Alternatively, I could just update the .cjsx to include `<ul>`'s. 

# Required Manual Testing

- [x] Does the non-logged in home page render correctly?
- [x] Does the logged in home page render correctly?
- [x] Does the projects page render correctly?
- [x] Can you load project home pages?
- [x] Can you load the classification page?
- [x] Can you submit a classification?
- [x] Does talk load correctly?
- [x] Can you post a talk comment?

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: css/secondary-page.styl [hunk 1]
```diff
@@ -55,11 +55,25 @@
...
+        li
+          margin: 0
+          padding: 0
+
         a
           @extends .main-nav-item
           font-size: 14px
           line-height: 3em
           text-decoration: none
+          &:focus
```
Comment (srallen) [inline css/secondary-page.styl]: Could you also apply these styles for hover? `&:focus, &:hover`

**Comment by srallen** [comment]
@lancesnider I agree updating the collections pages are out of scope of this PR

File: css/secondary-page.styl (hunk 1 above)
Comment (lancesnider) [reply css/secondary-page.styl]: Done. :) Sorry for the late reply. I've been trekking around the Colombian jungle. ;)

Comment (srallen) [reply css/secondary-page.styl]: No worries! I was on holiday last week too!

**Pull request merged**